packages/vite/src/runFeServer.ts

```diff
@@ -175,6 +175,9 @@ export async function runFeServer() {
     })
   )
 
+  // Serve static assets that aren't covered by any of the above routes or middleware
+  app.use(express.static(rwPaths.web.dist, { index: false }))
```

Comment:
Hey Josh, this looks fine! But could we also get it to ignore everything in the `server` directory?

I'm changing how this stuff will work for rsc builds anyway, but doesn't hurt to be thorough now.

Reply:
When you've implemented this nesting I think we will have an easier time only exposing what we intend to. For now, I need to expose it all and then block the server dist.

Reply:
Yes exactly.